src/main/resources/templates/application/summary/proceedings-and-costs-section.html

```diff
@@ -0,0 +1,163 @@
+      in the Case costs tab.
+    </p>
+    <div class="govuk-tabs" data-module="govuk-tabs">
+      <h2 class="govuk-tabs__title">
+        Proceedings, costs and prior authorities content
+      </h2>
+      <ul class="govuk-tabs__list">
+        <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
+          <a class="govuk-tabs__tab" href="#proceedings">
+            Proceedings
+          </a>
+        </li>
+        <li class="govuk-tabs__list-item">
+          <a class="govuk-tabs__tab" href="#case-costs">
+            Case costs
+          </a>
+        </li>
+        <li class="govuk-tabs__list-item">
+          <a class="govuk-tabs__tab" href="#prior-authority">
+            Prior authority
+          </a>
+        </li>
+      </ul>
+      <div class="govuk-tabs__panel" id="proceedings" aria-labelledby="tab_proceedings">
+        <h2 class="govuk-heading-l">Proceedings</h2>
+        <table class="govuk-table">
+          <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Matter type</th>
+            <th scope="col" class="govuk-table__header">Proceeding</th>
+            <th scope="col" class="govuk-table__header">Form of civil legal service</th>
+            <th scope="col" class="govuk-table__header">Client involvement type</th>
+            <th scope="col" class="govuk-table__header">Lead</th>
+            <th scope="col" class="govuk-table__header">Action</th>
+          </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+          <tr class="govuk-table__row" th:each="proceeding : ${proceedings.content}" th:unless="${proceedings.content.isEmpty()}">
+            <td class="govuk-table__cell">
+              <a th:href="@{/application/summary/proceedings/{id}/confirm(id=${proceeding.id})}"
+                 class="govuk-link" th:text="${proceeding.matterType.displayValue}"></a>
+            </td>
+            <td class="govuk-table__cell" th:text="${proceeding.proceedingType.displayValue}"></td>
+            <td class="govuk-table__cell" th:text="${proceeding.levelOfService.displayValue}"></td>
+            <td class="govuk-table__cell" th:text="${proceeding.clientInvolvement.displayValue}"></td>
+
+            <th:block th:if="${proceeding.leadProceedingInd}">
+              <td class="govuk-table__cell">
+                <img class="govuk-!-padding-left-2 govuk-!-padding-right-2" src="/tick.gif" alt="tick">
+              </td>
+              <td class="govuk-table__cell"></td>
+            </th:block>
+
+            <th:block th:unless="${proceeding.leadProceedingInd}">
+              <td class="govuk-table__cell"></td>
+              <td class="govuk-table__cell">
+                <a th:href="@{/application/summary/proceedings/{id}/remove(id=${proceeding.id})}"
+                   class="govuk-link" th:text="${'Delete'}">
```

Comment:
Any reason for these being thymeleaf text when they're static strings?

Reply:
amended